### ISSUE TITLE
Extra Curly Braces Outside of JSX

### ISSUE THREAD

**Issue by karlhorky** [meganrogge/template-string-converter#61]
In Template String Converter `0.5.8` it seems like there's a new bug which is causing extra curly braces to appear outside of JSX code (in a `.tsx` file with `TypeScript React` syntax):

https://user-images.githubusercontent.com/1935696/147680948-b533aadd-acd7-41b5-8e99-cd4267df9297.mp4

Minimal code to reproduce (with two examples):

```tsx
const name = ``; // <-- Add interpolation here

export default function Component() {
  const name2 = ``; // <-- Add interpolation here
  return <div />;
}
```

Maybe caused by features added for https://github.com/meganrogge/template-string-converter/issues/20 or https://github.com/meganrogge/template-string-converter/issues/15

**Comment by meganrogge** [comment]
@karlhorky Thanks for your continued reporting of issues with good repro cases. 

I do not use jsx props and it seems you have a vested interest in making sure the extension works well for them. I want this extension to work well, but have limited time at the moment.

Want to tweak this regex https://github.com/meganrogge/template-string-converter/blob/8aa1e44ffd2cdc65cacbd52f187845f760ca6521/src/extension.ts#L62

and make a PR?

**Comment by karlhorky** [comment]
Thanks for the fix in `0.5.9` @meganrogge !  🙌   Works well.

**Comment by imekachi** [comment]
Hi @meganrogge , I'm using `v0.6.1` and still have this issue with tsx files

https://github.com/meganrogge/template-string-converter/assets/5445243/a9aac46c-326a-466a-a7bf-c4c7ec560a3f

It also happens even when declaring a variable outside the component.
